defaults/main.yml

```diff
@@ -0,0 +1,16 @@
+# defaults file for setup_kdump
+kdump_package: kexec-tools
+kdump_conf_path: /etc/kdump.conf
+kdump_become_user: root
+
+kdump_set_crash_path: true
+kdump_crash_path: var/crash
+
+kdump_set_core_collector: false
```

Comment:
The user interface would be simpler if we could just check if the var is defined to populate the template with the arguments for all settings in this file.  @greg-hellings - Is our best practice to do this the way we did it in cinch? [1][2].  

In that case we left the vars commented out in _defaults/main.yml_ (but documented in the README.md) and then used an `omit` filter in the tasks.

- [1] https://github.com/RedHatQE/cinch/blob/master/cinch/roles/jenkins_common/defaults/main.yml#L8
- [2] https://github.com/RedHatQE/cinch/blob/master/cinch/roles/jenkins_common/tasks/main.yml#L35